types/chroma-js/index.d.ts

```diff
@@ -88,22 +88,22 @@ declare namespace chroma {
 
         /**
          * Mixes two colors. The mix ratio is a value between 0 and 1.
-         * The color mixing produces different results based the color space used for interpolation.
+         * The color mixing produces different results based the color space used for interpolation. Defaults to LRGB.
          * @example chroma.mix('red', 'blue', 0.25) // => #bf0040
          * @example chroma.mix('red', 'blue', 0.5, 'hsl') // => #ff00ff
          */
-        mix(color1: string | Color, color2: string | Color, f?: number, colorSpace?: keyof ColorSpaces): Color;
+        mix(color1: string | Color, color2: string | Color, f?: number, colorSpace?: InterpolationMode): Color;
```

Comment:
The docs call the `f` parameter `ratio` and the `colorSpace` parameter `mode`. Those seem like better names given this change. Approving, but suggesting that change if it makes sense to you.